.github/workflows/release.yml

```diff
@@ -67,8 +67,9 @@ jobs:
         run: |
           docker build \
             -t nipy/heudiconv:master \
+            -t nipy/heudiconv:unstable \
             -t nipy/heudiconv:latest \
-            -t nipy/heudiconv:"$(git describe)" \
+            -t nipy/heudiconv:"$(git describe | sed -e 's,^v,,g')" \
```

Comment:
```suggestion
            # remove "v" before version
            -t nipy/heudiconv:"$(git describe | sed -e 's,^v,,g')" \
```
optional nitpick

Reply:
that comment might break that `bash` multiline invocation so I better not risk it ;-)